frame/collective/src/lib.rs

```diff
@@ -183,7 +193,8 @@ pub mod pallet {
 		type Proposal: Parameter
 			+ Dispatchable<Origin = <Self as Config<I>>::Origin, PostInfo = PostDispatchInfo>
 			+ From<frame_system::Call<Self>>
-			+ GetDispatchInfo;
+			+ GetDispatchInfo
+			+ MaxEncodedLen;
```

Comment:
LOL dis fukn hard, I tried deriving `MaxEncodedLen` on each and every `Call` that gets generated, and we basically need to make sure that all call parameter types MUST also implement `MaxEncodedLen`, so no more arbitrary `Vec`s for dispatchable parameters anymore...